docs/guides/features/icons.md

```diff
@@ -1,28 +1,45 @@
+import Command from '@theme/Command'
+
 # Icons
 
-Tauri ships with a default iconset based on its logo. This is probably NOT what you want when you ship your application. To remedy this common situation, Tauri provides the `tauricon` command that will take an input file ("./app-icon.png" by default) and create all the icons needed for the various platforms:
+Tauri ships with a default iconset based on its logo. This is NOT what you want when you ship your application. To remedy this common situation, Tauri provides the `icon` command that will take an input file (`"./app-icon.png"` by default) and create all the icons needed for the various platforms:
 
 ## Usage
 
+Starting with `@tauri-apps/cli` / `tauri-cli` version 1.1 the `icon` is part of the main cli:
+
+<Command name="icon" />
+
 ```console
-npx @tauri-apps/tauricon --help
+> cargo tauri icon --help
+cargo-tauri-icon 1.1.0
 
-Description
-  Create all the icons you need for your Tauri app.
+Generates various icons for all major platforms
 
-  "ICON-PATH" is the path to the source icon (default: 'app-icon.png').
-  The icon needs to be either png (1240x1240 with transparency) or svg (square dimensions with transparency).
+USAGE:
+    cargo tauri icon [OPTIONS] [INPUT]
+
+ARGS:
+    <INPUT>    Path to the source icon (png, 1240x1240px with transparency) [default: ./app-icon.png]
+
+OPTIONS:
+    -h, --help               Print help information
+    -o, --output <OUTPUT>    Output directory. Default: 'icons' directory next to the tauri.conf.json file
+    -v, --verbose            Enables verbose logging
+    -V, --version            Print version information
+```
 
-Usage
-  npx @tauri-apps/tauricon [ICON-PATH]
+If you (temporarily) can't use Tauri CLI with version 1.1 or above you can use the deprecated `tauricon` package via `npx`:
```

Comment:
```suggestion
If you can't use Tauri CLI with version 1.1 or above you can use the deprecated `tauricon` package via `npx`:
```

Add a link to the repo?

Reply:
Also, this would be good to wrap in a :::note::: thingy

Reply:
really hesitant about this tbh, i want to promote the old package as little as possible. I guess the repo link is fair, but a note may highlight it too much 🤔

Reply:
I'd just remove this section completely then. Doesn't make sense to include non-supported things in the guides

Reply:
I'm fine with that if you don't tell on me x)

Reply:
I didn't see a thing 👀